src/photos/ducks/backup/components/BackupInfo.jsx

```diff
@@ -32,6 +54,33 @@ const BackupInfo = () => {
         </Alert>
       </div>
     )
+  } else if (status === 'done') {
```

Comment:
really nit: I was wondering whether removing a level of nesting wouldn't make it easier to read 🤔 
```js
} else if (status === 'done' && lastBackup.status === 'success') {
  return <BackupInfoSuccess />
} else if (status === 'done') {
  return <BackupInfoError />
}

return null
```